docs/install_guides/configurations-for-intel-gpu.md

```diff
@@ -13,10 +13,19 @@ If it is already installed, and you want to keep it, you can skip the installati
 
 ## Linux
 
+@sphinxdirective
+
+.. important:: The GPU plugin is supported on Ubuntu versions up to and including 20.04. Currently, the support on later releases of Ubuntu is still in experimental stages of development.
```

Comment:
why ? I don't agree with it. GPU plugin works on Ubuntu 22.04
Ubuntu 22.04 is compatible with Ubuntu 20.04, so it can run ANY binary which works on Ubuntu 20.04

What you want to write - is that we don't explicitly validate GPU on Ubuntu 22.04. Validation of GPU plugin on Ubuntu 22.04 implies validation of GPU driver on Ubuntu 22.04, but we don't perform validation of drivers / kernels / OS distributives.

Reply:
Agree with Ilya. That's absolutely not true. Moreover, certain HW, like Arctic Sound explicitly require Ubuntu 22.04, and they are validated.

Another question is what Ubuntu 22 is and how it differs from Ubuntu 20, i.e. Ubuntu 20.04.5 has kernel 5.15, and so does Ubuntu 22.04.0. I don't believe any of items from Ubuntu 22.04 changelog affects OpenVINO functionality.

Reply:
Thank you for explanation.

Reply:
#15440